extensions/amp-story/1.0/amp-story.js

```diff
@@ -1455,6 +1460,39 @@ export class AmpStory extends AMP.BaseElement {
     }
   }
 
+  /**
+   * @private
+   */
+  openMask_() {
+    const actions = Services.actionServiceForDoc(this.getAmpDoc());
+    if (!this.maskElement_) {
+      const mask = this.win.document.createElement('div');
```

Comment:
Nit: `maskElement` or `maskEl` (the styleguide recommends not using abbreviations, but `El` is a common exception to this rule)